spec/features/brexit_checker/email_signup_spec.rb

```diff
@@ -1,64 +1,68 @@
 require "spec_helper"
 require "gds_api/test_helpers/email_alert_api"
 
-RSpec.feature "Brexit Checker email signup", type: :feature do
-  include GdsApi::TestHelpers::ContentStore
-  include GdsApi::TestHelpers::EmailAlertApi
+unless ENV["FEATURE_FLAG_ACCOUNTS"] == "enabled"
 
-  let(:subscriber_list) do
-    {
-      "title" => "Get ready for 2021",
-      "slug" => "your-get-ready-for-brexit-results-a1a2a3a4a5",
-      "description" => "[You can view a copy of your results on GOV.UK.](https://www.test.gov.uk/transition-check/results?c%5B%5D=nationality-eu)",
-      "tags" => { "brexit_checklist_criteria" => { "any" => %w[nationality-eu] } },
-      "url" => "/transition-check/results?c%5B%5D=nationality-eu",
-      "group_id" => BrexitCheckerController::SUBSCRIBER_LIST_GROUP_ID,
-    }
-  end
+  RSpec.feature "Brexit Checker email signup", type: :feature do
```

Comment:
This all should be unchanged just nested under the feature flag... annoying big diff...